src/python/pants/backend/jvm/tasks/scalafix.py

```diff
@@ -50,9 +50,30 @@ def source_extension(cls):
   @classmethod
   def prepare(cls, options, round_manager):
     super().prepare(options, round_manager)
+    round_manager.require_data('zinc_args') 
```

Comment:
The `zinc_args` product is produced by compilation, similar to the `runtime_classpath` product. So this needs to be under the `if options.semantic` check.

This is causing the CI failure.